### DISCUSSION TITLE
Is it possible for a mac user to run the windows app on mac for development purposes?

### DISCUSSION THREAD

**Discussion by DevVibhor** [microsoft/react-native-windows#12437] [General]
I don't have a windows machine, my react-native app works fine on iOS, Android and macOS but I am unable to check my app for windows. Can anyone please recommend something that will be of great help for me?

**Comment by ghost** [comment]
I think this topic answers your question: https://microsoft.github.io/react-native-windows/docs/win10-vm 🤔

**Reply by DevVibhor** [reply]
Hey @Victor0814gui thanks for sharing the link. I somehow missed it in the documentation.

**Comment by ghost** [comment]
I confess I get lost in the documentation 😅, but did the comment above solve your issue @DevVibhor?

**Reply by DevVibhor** [reply]
Hi @Victor0814gui I haven't tried the solution yet but I will most probably be going with Local VM to begin with.
